File: 源码/lh_DroidPlugin/app/src/main/java/com/enjoy/lh_droidplugin/MainActivity.java
package com.enjoy.lh_droidplugin;

import androidx.appcompat.app.AppCompatActivity;

import android.content.ComponentName;
import android.content.Intent;
import android.os.Bundle;
import android.os.RemoteException;
import android.util.Log;
import android.view.View;

import com.morgoo.droidplugin.pm.PluginManager;

public class MainActivity extends AppCompatActivity {

    private final static String apkPath = "/sdcard/plugin-debug.apk";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btn1).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    PluginManager.getInstance().installPackage(apkPath, 0);
                    Log.e("leo", "完成插件的加载");
                } catch (RemoteException e) {
                    e.printStackTrace();
                }
            }
        });

        findViewById(R.id.btn2).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    Intent intent = new Intent();
                    intent.setComponent(new ComponentName("com.enjoy.plugin", "com.enjoy.plugin.MainActivity"));
                    startActivity(intent);
                } catch (Exception e) {
                    Log.e("leo", "加载出错了");
                    e.printStackTrace();
                }
            }
        });

        findViewById(R.id.btn3).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    PluginManager.getInstance().deletePackage("com.enjoy.plugin", 0);
                } catch (RemoteException e) {
                    e.printStackTrace();
                }
            }
        });

    }

}

File: android/ThemeDemo/app/src/main/java/com/joyy/themedemo/MainActivity.kt
package com.joyy.themedemo

import android.app.Activity
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.TypedValue
import android.view.View
import android.widget.TextView
import androidx.core.content.ContextCompat

/**
 * https://blog.csdn.net/yingtian648/article/details/106528885
 */
class MainActivity : AppCompatActivity() {
    companion object {
        var currentStyle: Int = R.style.Theme_ThemeDemo_BLACK

        fun setMyTheme(act: Activity, style: Int) {
            currentStyle = style
            act.setTheme(currentStyle)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        // Theme.ThemeDemo.BLACK
        setTheme(R.style.Theme_ThemeDemo_BLACK)
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val findViewById = findViewById<TextView>(R.id.txt)
        findViewById.text = "MainActivity"
        findViewById.setTextColor(ContextCompat.getColor(this, R.color.aaaa))


        findViewById<View>(R.id.btn).setOnClickListener {
            startActivity(Intent(this, SecondActivity::class.java))
        }

        findViewById<View>(R.id.btnBlack).setOnClickListener {
            currentStyle = R.style.Theme_ThemeDemo_BLACK
            setTheme(currentStyle)
        }
        findViewById<View>(R.id.btnRed).setOnClickListener {
            currentStyle = R.style.Theme_ThemeDemo_RED
            setTheme(currentStyle)

        }
        findViewById<View>(R.id.btnGreen).setOnClickListener {
            currentStyle = R.style.Theme_ThemeDemo_RED
            setTheme(currentStyle)
        }
    }

}